server/routes/api/documents.ts

```diff
@@ -91,7 +91,7 @@ router.post("documents.list", auth(), pagination(), async (ctx) => {
     // index sort is special because it uses the order of the documents in the
     // collection.documentStructure rather than a database column
     if (sort === "index") {
-      documentIds = collection.documentStructure
+      documentIds = (collection.documentStructure || [])
```

Comment:
did something change that allows document structure to come back as null or undefined?

Reply:
An existing bug I could pull into it's own PR, if you try and do `documents.list` for a new and empty collection you'll get an internal server error today